src/Components/CategoryCard/style.component.css

```
@@ -0,0 +1,140 @@
+.category-card{
+    width: 12.5rem;
+    height: 16rem;
+    overflow: hidden;
+    filter: drop-shadow(2px 4px 4px rgba(0, 0, 0, 0.3));
+    border-radius: 50px ;
+    background-color: var(--primary-color);
+    position: relative;
+    margin: 100px 100px 100px 100px;
```

Colocar margin: 100px já aplica o margin para todos os cantos
```suggestion
    margin: 100px;
```